src/dashboard/Push/PushNew.react.js

```diff
@@ -670,21 +714,40 @@ export default class PushNew extends DashboardView {
       {this.renderExperimentContent(fields, setField)}
     </Fieldset> : null;
 
+    const richmediaFieldsLegend = PARSE_SERVER_SUPPORTS_PUSH_RICH_MEDIA ?
+      'Rich media push notification' :
+      'Rich notifications';
+
+    const richmediaFieldsDescription = PARSE_SERVER_SUPPORTS_PUSH_RICH_MEDIA ?
+      'We can send images and videos directly to your app.' :
+      "If your push hasn't been send with rich media, it won't get setup by your dev.";
+
+    const richmediaTimeFields = PARSE_SERVER_SUPPORTS_PUSH_RICH_MEDIA? <Fieldset
+      legend={richmediaFieldsLegend}
+      description={richmediaFieldsDescription}>
+      <Field
+          label={<Label text='Send media attachment?' />}
+          input={<Toggle value={fields.media_attachment} onChange={setField.bind(null, 'media_attachment')} />} />
+      {PARSE_SERVER_SUPPORTS_PUSH_RICH_MEDIA ? this.renderRichMediaContent(fields, setField) : null}
+    </Fieldset> : null;
+
     const timeFieldsLegend = PARSE_SERVER_SUPPORTS_SCHEDULE_PUSH ?
       'Choose a delivery time' :
       'Choose exiry';
 
     const timeFieldsDescription = PARSE_SERVER_SUPPORTS_SCHEDULE_PUSH ?
-      'We can send the campaign immediately, or any time in the next 2 weeks.' :
+      'We can send the campaign immediately, or any time in the next weeks.' :
```

Comment:
Should this be changed to `We can send the campaign immediately, or any time in the future.`?